native/src/routes/__tests__/SearchModal.spec.tsx

```diff
@@ -25,7 +26,7 @@ describe('SearchModal', () => {
   })
   const dummy = jest.fn()
 
-  const t = (key: string) => key
+  const t = ((key: string) => key) as TFunction
```

Comment:
Optional: Otherwise we could also just not pass tFunction as prop but use the hook instead